docker/config-discovery/config-server.py

```diff
@@ -27,7 +27,7 @@ def make_configs(network):
             if config1 == config2:
                 continue
 
-            c['group'].append(extract(config2))
+#            c['group'].append(extract(config2))
```

Comment:
When using `docker-compose` for add-peer-test, I erase this line. because, result in it add another peer.( in other word, it can not do add-peer-test )
But, I don't use `docker-compose` for do add-peer-test. I will restore.